.github/CODEOWNERS

```diff
@@ -1799,7 +1799,7 @@
 /types/expo-mixpanel-analytics/                         @martintreurnicht
 /types/expo__status-bar-height/                         @dawnmist
 /types/express/                                         @borisyankov @CMUH @puneetar @dfrankland
-/types/express-actuator/                                @etruta
+/types/express-actuator/                                @etruta @rcruzper
```

Comment:
```suggestion
/types/express-actuator/                                @ethanrubio
```

Reply:
oh, sorry I didn't understand the documentation, fixed.